### PR TITLE
[update] Connect TimescaleDB and Grafana

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

The install link to Grafana in the tutorials was to the install instructions in the [Grafana documentation](https://grafana.com/docs/grafana/v9.0/setup-grafana/installation/)

Updated the install link to [Grafana service on MST](https://docs.timescale.com/timescaledb/latest/tutorials/grafana/installation/#create-a-new-service-for-grafana)

This is to ensure that the content of the tutorial can optimise signups.

I also tested connecting the Timescale Cloud instance to Grafana on MST service and it works.
Screenshot 2022-08-19 at 8 44 05 PM
Screenshot 2022-08-19 at 8 44 18 PM

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: timescaledb/tutorials/grafana/grafana-timescalecloud.md [hunk 1]
```diff
@@ -7,18 +7,21 @@ keywords: [Grafana, visualizations, analytics]
...
 # Connect TimescaleDB and Grafana
+
 Grafana ships with built-in Prometheus, PostgreSQL, Jaeger, and other data
 source plugins that allow you to query and visualize data from a compatible
 database. To add a data source in Grafana you must be signed as a user with
 organization administration role privileges.
 
-To connect Grafana with TimescaleCloud database instance, install Grafana. For
-information about installing Grafana, see the [Grafana installation documentation][grafana-install].
+To connect Grafana with TimescaleCloud database instance, create a
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/grafana-timescalecloud.md:16]:
```suggestion
To connect Grafana with your Timescale Cloud database, create a
```

File: timescaledb/tutorials/grafana/grafana-timescalecloud.md [hunk 2]
```diff
@@ -7,18 +7,21 @@ keywords: [Grafana, visualizations, analytics]
...
+[Grafana][install-grafana] service on [MST][mst-login]. You can try it for
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/grafana-timescalecloud.md:17]: Wait - Cloud or MST?

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: timescaledb/tutorials/grafana/grafana-timescalecloud.md (hunk 2 above)
Comment (Rajakavitha1) [reply timescaledb/tutorials/grafana/grafana-timescalecloud.md:17]: This is about the [Grafana service on MST](https://docs.timescale.com/timescaledb/latest/tutorials/grafana/installation/#create-a-new-service-for-grafana)

Comment (Loquacity) [reply timescaledb/tutorials/grafana/grafana-timescalecloud.md:17]: In that case, you need to say MST on L16.

Comment (Rajakavitha1) [reply timescaledb/tutorials/grafana/grafana-timescalecloud.md:17]: > In that case, you need to say MST on L16.

Yes!!! makes sense. Thanks for pointing it.

File: timescaledb/tutorials/grafana/grafana-timescalecloud.md [hunk 3]
```diff
@@ -7,18 +7,21 @@ keywords: [Grafana, visualizations, analytics]
...
 # Connect TimescaleDB and Grafana
+
 Grafana ships with built-in Prometheus, PostgreSQL, Jaeger, and other data
 source plugins that allow you to query and visualize data from a compatible
 database. To add a data source in Grafana you must be signed as a user with
 organization administration role privileges.
 
-To connect Grafana with TimescaleCloud database instance, install Grafana. For
-information about installing Grafana, see the [Grafana installation documentation][grafana-install].
+To connect Grafana service on MST with your Timescale Cloud database, create a
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/grafana-timescalecloud.md:16]: You can't have both MST and Timescale Cloud.

```suggestion
To connect Grafana with your Managed Service for TimescaleDB database, create a
```

Comment (Rajakavitha1) [reply timescaledb/tutorials/grafana/grafana-timescalecloud.md]: Hi @Loquacity  I reworked on this PR and added Grafana service on MST as an alternative option. I hope this draft removes the ambiguity.

File: timescaledb/tutorials/grafana/grafana-timescalecloud.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ as data source in [Grafana][grafana-homepage].
 
 ## Configure TimescaleDB as a data source
 
-To configure TimscaleDB database as a data source you need to create a service,
+To configure TimscaleDB as a data source you need to create a service,
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/grafana-timescalecloud.md:28]:
```suggestion
To configure TimescaleDB as a data source you need to create a service,
```

Ugh, only just spotted that one, sorry!

File: timescaledb/tutorials/grafana/grafana-timescalecloud.md [hunk 5]
```diff
@@ -7,21 +7,26 @@ keywords: [Grafana, visualizations, analytics]
...
 # Connect TimescaleDB and Grafana
+
 Grafana ships with built-in Prometheus, PostgreSQL, Jaeger, and other data
 source plugins that allow you to query and visualize data from a compatible
 database. To add a data source in Grafana you must be signed as a user with
 organization administration role privileges.
 
-To connect Grafana with TimescaleCloud database instance, install Grafana. For
-information about installing Grafana, see the [Grafana installation documentation][grafana-install].
+To connect Grafana with TimescaleDB service on Timescale Cloud, install Grafana.
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/grafana-timescalecloud.md:16]:
```suggestion
To connect Grafana with TimescaleDB on Timescale Cloud, start by installing Grafana.
```

File: timescaledb/tutorials/grafana/grafana-timescalecloud.md [hunk 6]
```diff
@@ -7,21 +7,26 @@ keywords: [Grafana, visualizations, analytics]
...
+For information about installing Grafana, see the [Grafana installation documentation][grafana-install].
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/grafana-timescalecloud.md:17]:
```suggestion
For more information about installing Grafana, see the [Grafana installation documentation][grafana-install].
```

File: timescaledb/tutorials/grafana/grafana-timescalecloud.md [hunk 7]
```diff
@@ -7,21 +7,26 @@ keywords: [Grafana, visualizations, analytics]
...
+
+Alternatively, to connect Grafana service on MST with your TimescaleDB service
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/grafana-timescalecloud.md:19]:
```suggestion
Alternatively, to connect your Grafana service to MST with your TimescaleDB service
```

File: timescaledb/tutorials/grafana/grafana-timescalecloud.md [hunk 8]
```diff
@@ -7,21 +7,26 @@ keywords: [Grafana, visualizations, analytics]
...
+on Timescale Cloud, create a [Grafana][install-grafana] service on
+[MST][mst-login]. You can try it for free for 30 days.
 
-This section shows you how to connect TimescaleCloud database instance as data
-source in [Grafana][grafana-homepage].
+This section shows you how to connect TimescaleDB service on Timescale Cloud
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/grafana-timescalecloud.md:23]:
```suggestion
This section shows you how to connect TimescaleDB on Timescale Cloud
```

File: timescaledb/tutorials/grafana/grafana-timescalecloud.md [hunk 9]
```diff
@@ -7,21 +7,26 @@ keywords: [Grafana, visualizations, analytics]
...
+as data source in [Grafana][grafana-homepage].
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/grafana-timescalecloud.md:24]:
```suggestion
as a data source in [Grafana][grafana-homepage].
```

Comment (Loquacity) [reply timescaledb/tutorials/grafana/grafana-timescalecloud.md:24]: I wonder if a use case example would work here, to try and make it clearer?

File: timescaledb/tutorials/grafana/grafana-timescalecloud.md [hunk 10]
```diff
@@ -41,6 +46,15 @@ Grafana.
 
 ### Configuring TimescaleDB as data source
 
+To configure TimescaleDB service on Timescale Cloud with your Grafana
+installation, log in to Grafana and proceed to step 5 in this procedure.
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/grafana-timescalecloud.md:50]:
```suggestion
```

I think this is just confusing.

**Review by Loquacity — CHANGES_REQUESTED** [review]

**Comment by charislam** [comment]
I wonder if there is an actual use case for this. Does anyone want two TimescaleDB Cloud services, on two different platforms? Might be a question worth taking to Developer Advocacy.

Otherwise LGTM

**Review by Loquacity — APPROVED** [review]

**Comment by Loquacity** [comment]
@Rajakavitha1 What's the status on this?

**Comment by Rajakavitha1** [comment]
> @Rajakavitha1 What's the status on this?

We need a tech approval from @mathisve !!!!

**Comment by mathisve** [comment]
I agree with @Loquacity changes. 

Further it might be worth mentioning something about the TLS/SSL options (and why you can't use them yet with Timescale Cloud). We had a comment about this on a YouTube video: https://iobeam.slack.com/archives/CPEERKKJL/p1659460853865649

Other than that, looks good to me!

**Review by mathisve — APPROVED** [review]

**Comment by Rajakavitha1** [comment]
> https://iobeam.slack.com/archives/CPEERKKJL/p1659460853865649

Thanks @mathisve !!! We have mentioned about the TLS to require in the document.

**Pull request merged**